africanus/gps/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think the top level __init__.py got cut and pasted here. Rather just leave it empty.